src/Controls/src/Core.Design/AttributeTableBuilder.cs

```diff
@@ -30,6 +30,9 @@ public AttributeTableBuilder()
 			AddMemberAttributes("Microsoft.Maui.Controls.VisualElement", "Visual",
 			   new TypeConverterAttribute(typeof(VisualDesignTypeConverter)));
 
+			AddMemberAttributes("Microsoft.Maui.Controls.VisualElement", "FlowDirection",
```

Comment:
side question, why do we have a [TypeConverter] on the Property, and not on the Type ?

Reply:
Probably because the real type converter is also on the property.

Reply:
> Probably because the real type converter is also on the property.

that was my question. Why are we doing this at the Core and Controls level. I understand we have to mimic it here

Reply:
Yeah, that I don't know. @Redth?

Reply:
Not sure, I was mostly just following what had been done for similar cases previously.

Reply:
What this means is that the FlowDirection enum will only behave this way for the VisualElement.FlowDirection property. If that type is used for any other property, by 3rd party control vendors for example, then it will behave just like the regular enum and not allow those CSS values to be used.